### PR TITLE
fix: valid google URL not being accepted

### PR DESCRIPTION
**Pull request by BobbyRobBob**
- [ ] PR title follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specifications
- [ ] PR assignee has been selected
- [ ] PR label has been selected

## 🔧 What changed
Updated the form validation for google URL to accept google maps short format.

## 🧪 Testing instructions
-Run yarn dev:localserver
-Click on "Add a doctor"
-Enter short form google maps link

Expected results: Short link will not trigger validation

## 📸 Screenshots

- ### Before
image

- ### After
image

## Summary by CodeRabbit

* **Bug Fixes**
  * Strengthened validation logic for Google Maps URLs to properly recognize and validate additional URL formats and patterns, providing more reliable input handling across different submission scenarios.

* **Tests**
  * Expanded test coverage for Google Maps URL validation to include edge cases and various URL format variations.

### CODE REVIEW

File: utils/formValidations.ts [hunk 1]
```diff
@@ -218,11 +218,8 @@ export function validateUserSubmittedFirstName(name: string): boolean {
 
 export function validateGoogleMapsUrlInput(url: string): boolean {
     url = url.trim()
-    if (url.startsWith('https://www.google.com/maps') || url.startsWith('https://www.google.co.jp/maps')
-      || url.startsWith('https://maps.google.com/')) {
-        return true
-    }
-    return false
+    const googleMapsRegex = /^https:\/\/(www\.)?(google\.com\/maps|google\.co\.jp\/maps|maps\.google\.com|maps\.app\.goo\.gl)\//
+    return googleMapsRegex.test(url)
```
[the remark this answers is not in the document]
Comment (NabbeunNabi) [reply utils/formValidations.ts:222]: @BobbyRobBob I agree with this

**Review by NabbeunNabi — CHANGES_REQUESTED** [review]
I would follow up on coderabbit's comments and also if you could left align the error message here as well
